src/main/java/com/notably/logic/commands/OpenCommand.java

```diff
@@ -26,6 +30,7 @@ public OpenCommand(AbsolutePath path) {
     @Override
     public void execute(Model notablyModel) throws CommandException {
         requireNonNull(notablyModel);
+        logger.info("Executing OpenCommand");
         try {
             notablyModel.setCurrentlyOpenBlock(path);
         } catch (NoSuchBlockException ex) {
```

Comment:
Let's add another `logger.info()` after the `try-catch` execution? Just to be consistent with the other commands' logging